tests/models_tests/test_scan.py

```diff
@@ -257,6 +254,20 @@ def test__nifti_before_dicom_to_nifti(self):
         result = self.scan._nifti
         self.assertIsNone(result)
 
+    def test_suggest_subject(self):
+        self.scan.suggest_subject(self.subject)
+        self.assertEqual(self.scan.session.subject, self.subject)
+
+    def test_suggest_subject_mismatch(self):
+        subject = Subject.objects.create()
+        self.scan.suggest_subject(subject)
+        self.assertEqual(self.scan.session.subject, self.subject)
+
+    def test_suggest_subject_none(self):
+        subject = None
+        self.scan.suggest_subject(subject)
+        self.assertEqual(self.scan.session.subject, self.subject)
+
```

Comment:
Check if `suggest_subject()` is really required,  if not remove it and these tests :(